lib/mv_opentelemetry/absinthe.ex

```diff
@@ -47,13 +46,12 @@ defmodule MvOpentelemetry.Absinthe do
     ]
   end
 
-  def handle_start_event([:absinthe, :resolve, :field, :start], _measurements, meta, _opts) do
-    event_name = Enum.join([:absinthe, :resolve, :field], ".")
+  def handle_start_event([:absinthe, :resolve, :field, :start], _measurements, meta, opts) do
+    event_name = Enum.join(opts[:name_prefix] ++ [:resolve, :field], ".")
 
     resolution = meta.resolution
 
     attributes = [
-      "graphql.field.arguments": resolution.arguments,
```

Comment:
I removed the arguments here, we might be logging sensitive data otherwise, and since this is a map, we cannot guarantee that we won't leak anything weird.